lsst/ts/cbp/statemachine.py

```diff
@@ -424,8 +301,8 @@ class CBPModel:
     parked: int
         The last updated parked attribute.
     """
-    def __init__(self,port: str, address: int):
-        self._cbp = CBPComponent(port, address)
+    def __init__(self):
+        self._cbp = component.CBPComponent()
```

Comment:
Please consider *not* copying all those attributes from the model (e.g. self.mask = self._cbp.mask). Instead just ask for it in the _cbp. Duplicating data tends to lead to hard-to-diagnose errors and complicates the interface between the CSC and the model. Let the model keep track of the data and leave the CSC to just deal with forwarding commands and reporting state changes.

I suggest you make the model a public attribute of the CSC by naming it `cbp` instead of `_cbp`, and always use the model when you can.

Even the `connect` and `disconnect` methods don't need to be in the CSC and probably should not be. Just call the version in the model when you appropriate. (You have to forward the `configure` method, but that is probably one of the few bits of required duplication.)